ddocs/medic-client/views/people_by_phone/map.js

```diff
@@ -1,5 +1,5 @@
 function(doc) {
-  if (doc.type === 'person') {
-    emit([doc.phone]);
+  if (doc.phone) {
```

Comment:
I've modified the view to emit only contacts with a phone and renamed it to `contacts_by_phone`.